packages/ts/lit-form/src/BinderNode.ts

```diff
@@ -202,12 +300,12 @@ export class BinderNode<M extends AbstractModel = AbstractModel> extends EventTa
    * attribute on the field components.
    */
   get name(): string {
-    let { model }: { model: AbstractModel } = this;
+    let { model }: { model: ProvisionalModel } = this;
     let name = '';
 
-    while (model[_parent] instanceof AbstractModel) {
-      name = `${String(model[_key])}${name ? `.${name}` : ''}`;
-      model = model[_parent];
+    while (model[$owner] instanceof AbstractModel || model[$owner] === Model || model[$owner] instanceof Model) {
```

Comment:
```suggestion
    while (model[$owner] instanceof AbstractModel || model[$owner] instanceof Model) {
```
You don't have to separately check if it is a `Model`. `instanceof` should do it for you.